docs/usage/.pages

```diff
@@ -1,5 +1,6 @@
 nav:
   - Home: 'index.md'
+  - Reading List: 'reading-list.md'
```

Comment:
By the way, this change will break a Cypress test on the Renovate docs repository. We have a test that checks if the next/previous buttons work properly. We hard-coded the `getting-started` page as the Next page.

Reply:
can be fixed later

Reply:
We'll have to fix it later when the docs change lands in the Renovate docs repository. I can't fix it now. 😄 

Now you know _why_ and _how_ the test is going to break, so you can fix it if I'm not around. 😉